chart/values.yaml

```
@@ -919,19 +929,22 @@ config:
     colored_console_log: 'False'
   metrics:
     statsd_on: '{{ ternary "True" "False" .Values.statsd.enabled }}'
-    statsd_port: 9125
+    statsd_port: '{{ .Values._ports.statsdIngest }}'
```

```suggestion
    statsd_port: '{{ .Values.ports.statsdIngest }}'
```

This should be the service port as Airflow will go through the service.